test/integration/retry_admission_control/config.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto3";
```

Comment:
I added a integration test toy extension to be able to validate the basics in a test. This toy extension either allows all retries or denies all retries.